src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -431,6 +432,14 @@ def lvm_group_name(device)
       def fixed_size?(device)
         device.is?(:disk_device, :software_raid)
       end
+
+    private
+
+      def include_partition_type?(partition)
```

Comment:
np: The name is confusing to me. I read that I am setting the `@partition_type` to `primary` if the section includes a partition type (which is not exactly what you mean). I would prefer something like `primary_partition?` or something like that.